packages/clerk-js/src/core/clerk.ts

```diff
@@ -2010,8 +2006,12 @@ export class Clerk implements ClerkInterface {
    * The handshake payload is transported in the URL in development. In cases where FAPI is returning the handshake payload, but Clerk is being used in a client-only application,
    * we remove the handshake associated parameters as they are not necessary.
    */
-  #clearHandshakeFromUrl = () => {
+  #clearClerkQueryParams = () => {
     try {
+      removeClerkQueryParam(CLERK_SYNCED);
+      // @nikos: we're looking into dropping this param completely
+      // in the meantime, we're removing it here to keep the URL clean
+      removeClerkQueryParam(CLERK_SUFFIXED_COOKIES);
```

Comment:
This should never make it to the client, I'm pretty sure it being added to the redirect is a bug in FAPI that needs to be fixed.